### PR TITLE
Be more pythonic

### PR DESCRIPTION
**Pull request by tetienne**
I have 
* rename all the functions from CamelCase to snake case
* replace the getter functions by properties
* apply [black](https://github.com/psf/black) formatting 
* use list comprehension when possible

I haven't a opensprinkler device, so I cannot test locally.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1 @@
-httplib2==0.10.3
```
Comment (vinteo) [inline requirements.txt:1]: should these be removed?

**Comment by vinteo** [comment]
Thanks for this, I come from a javascript background hence all the camel case lol. I will test it when I get a chance

File: requirements.txt (hunk 1 above)
Comment (tetienne) [reply requirements.txt:1]: No, but as you write it in the setup.py, you can avoid to repeat yourself with the -e .
Here in the requirements.txt, add only the dev dependencies, like pytest for instance.

I suggest you to look at [poetry](https://python-poetry.org/) to really ease the packaging of your library.

**Comment by tetienne** [comment]
You are welcome. I hesitated with Java 😊

File: pyopensprinkler/station.py [hunk 2]
```diff
@@ -22,44 +22,50 @@ def index(self):
 
     def _getStatusVariable(self, statusIndex):
```
Comment (vinteo) [inline pyopensprinkler/station.py:23]: missed this one

File: pyopensprinkler/station.py [hunk 3]
```diff
@@ -22,44 +22,50 @@ def index(self):
...
         """Retrieve seconds remaining"""
         return self._getStatusVariable(1)
 
-    def getStatus(self):
+    @property
+    def status(self):
         """Retrieve status"""
-        isRunning = self.getIsRunning()
-        pid = self.getRunningProgramId()
+        is_running = self.is_running()
```
Comment (vinteo) [inline pyopensprinkler/station.py:55]: this is a property now

**Comment by vinteo** [comment]
found a couple of issues while testing but will merge and fix immediately

**Pull request merged**